tracer/test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -1080,6 +1080,11 @@ private void HandleHttpRequests()
 
                             var mockTracerResponse = HandleHttpRequest(MockHttpParser.MockHttpRequest.Create(ctx.Request));
 
+                            if (!mockTracerResponse.SendResponse)
+                            {
+                                ctx.Response.Abort(); // close without sending to avoid getting blocked for 15 seconds
+                            }
+
```

Comment:
This one was causing a 15 second block on `GetContext()` here for when we were testing TCP with no response from the agent.

With this, instead of keeping the response open I opted to just abort it (still doesn't send it), but I'd be willing to revert this is this doesn't seem like a good idea. It seemed to impact 2 tests, so not a major one.

Reply:
This only applies to the agent malfunction tests, right? I think this is a reasonable approach to take to avoid such long waits

Reply:
Ah yeah, this was only for those two tests for the agent malfunction tests